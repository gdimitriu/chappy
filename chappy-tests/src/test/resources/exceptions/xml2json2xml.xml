<?xml version="1.0"?>
<flow>
   <steps>
      <step class="Xml2JsonStep">
         <disabled>true</disabled>
         <parameters1>
         	<mode>xml2json</mode>
         </parameters1>
      </step>
      <step class="Json2XmlStep">
         <disabled>false</disabled>
         <parameters>
         	<mode>json2xml</mode>
         </parameters>
      </step>
   </steps>
</flow>
