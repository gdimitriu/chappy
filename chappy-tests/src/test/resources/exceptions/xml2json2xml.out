javax.xml.bind.UnmarshalException
 - with linked exception:
[org.xml.sax.SAXParseException; lineNumber: 6; columnNumber: 23; cvc-complex-type.2.4.a: Invalid content was found starting with element 'parameters1'. One of '{configuration, parameters}' is expected.]